common-react/@dbeaver/ui-kit/src/index.ts

```diff
@@ -21,3 +21,15 @@ export {
   type SelectItemProps,
 } from './Select/Select.js';
 export { SelectField, type SelectFieldProps, type SelectItem } from './Select/SelectField.js';
+export {
+  Radio,
+  RadioGroup,
+  _Radio,
+  _RadioGroup,
+  _RadioProvider,
```

Comment:
if we don't use it right now it's better to not export inner primitives from ariakit